gum/backend-x86/gumstalker-x86.c

```diff
@@ -2601,6 +2608,49 @@ gum_exec_block_backpatch_inline_cache (GumExecBlock * block,
   }
 }
 
+
+#if GLIB_SIZEOF_VOID_P == 4 && defined (HAVE_WINDOWS)
+static GumVirtualizationRequirements
+gum_exec_block_virtualize_wow64_transition (GumExecBlock * block,
```

Comment:
Let's move this right after `gum_exec_block_virtualize_sysenter_insn()`, and add a forward declaration matching the order.